src/deluge/gui/ui/keyboard/layout/isomorphic.cpp

```diff
@@ -20,35 +20,100 @@
 #include "gui/ui/audio_recorder.h"
 #include "gui/ui/browser/sample_browser.h"
 #include "gui/ui/sound_editor.h"
+#include "model/instrument/melodic_instrument.h"
+#include "model/model_stack.h"
 #include "model/settings/runtime_feature_settings.h"
 #include "util/functions.h"
 #include <limits>
 
+#define VEL_COL kDisplayWidth
+#define MOD_COL (kDisplayWidth + 1)
+
 namespace deluge::gui::ui::keyboard::layout {
 
+static inline void popupUint8(deluge::hid::Display* display, uint8_t val) {
```

Comment:
This function is useful enough that I think it should live in the display class for convenience